torch/optim/__init__.py

```diff
@@ -13,6 +13,7 @@
 from torch.optim.adam import Adam as Adam
 from torch.optim.adamax import Adamax as Adamax
 from torch.optim.adamw import AdamW as AdamW
+from torch.optim.ademamix import AdEMAMix as AdEMAMix
```

Comment:
I think this could be rewritten to just:

`from torch.optim.ademamix import AdEMAMix`